packages/sails-ng-common/src/validation/validators-support.ts

```diff
@@ -1,21 +1,28 @@
-import {FormValidatorConfig, FormValidatorDefinition, FormValidatorFn} from "./form.model";
+import {
+    FormValidationGroups,
+    FormValidatorConfig,
+    FormValidatorDefinition,
+    FormValidatorFn
+} from "./form.model";
 
 export class ValidatorsSupport {
     /**
      * Create validator definition mapping from the validator definitions.
      * @param definition The form validator definitions.
      */
-    public createValidatorDefinitionMapping(definition: FormValidatorDefinition[] | null | undefined) {
+    public createValidatorDefinitionMapping(
+        definition: FormValidatorDefinition[] | null | undefined
+    ): Map<string, FormValidatorDefinition> {
         const defMap = new Map<string, FormValidatorDefinition>();
         for (const definitionItem of (definition ?? [])) {
-            const name = definitionItem?.name;
+            const validatorClass = definitionItem?.class;
```

Comment:
Probably should check the validatorClass is valid? Something like this.
```suggestion
            const validatorClass = definitionItem?.class;
            if (!validatorClass || typeof validatorClass !== 'string' || validatorClass.trim() === '') {
               throw new Error(`Validator definition must have a valid 'class' property. `
                    + `Received: '${validatorClass}', message: '${definitionItem?.message}'.`);
            }
```